src/javascript/app/pages/resources/metatrader/download.ui.js

```diff
@@ -26,6 +26,8 @@ const toggleDownloadPage = target => {
         getElementById(`${target}-description`).setVisibility(1);
     }
     getElementById(`mt5_download_${target === 'mac' ? 'mac_' : ''}platforms`).setVisibility(1);
+    getElementById(`mt5_download_${target !== 'mac' ? 'mac_' : ''}platforms`).setVisibility(0);
```

Comment:
Don't both of these lines mean the same?

Reply:
No, second string resolves the component duplication problem. 
![image](https://user-images.githubusercontent.com/78138881/106872435-59cb3b80-66e4-11eb-96f4-7c20dbd3078b.png).   What happens without it